tests/appinfo/routes.php

```diff
@@ -85,5 +85,6 @@
 		['name' => 'Settings#deprecatedRouteAndParameterGet', 'url' => '/api/{apiVersion}/deprecated-route-parameter-get', 'verb' => 'GET', 'requirements' => ['apiVersion' => '(v2)']],
 		['name' => 'Settings#samePathGet', 'url' => '/api/{apiVersion}/same-path', 'verb' => 'GET', 'requirements' => ['apiVersion' => '(v2)']],
 		['name' => 'Settings#samePathPost', 'url' => '/api/{apiVersion}/same-path', 'verb' => 'POST', 'requirements' => ['apiVersion' => '(v2)']],
+		['name' => 'V1\SubDir#subDirRoute', 'url' => '/sub-dir', 'verb' => 'GET'],
```

Comment:
Windows style paths in the name?

Reply:
I wasn't even aware that subfolders worked

Reply:
> I wasn't even aware that subfolders worked

Me neither until yesterday :see_no_evil: 

> Windows style paths in the name?

No, it's the backslash from the namespace. Routes never operate based on the path but the namespace (but of course those are very closely related with PSR-4).